src/subsplit_dag.cpp

```diff
@@ -1095,18 +1116,23 @@ SubsplitDAG::ModificationResult SubsplitDAG::AddNodePair(const Bitset &parent_su
     // Reindex these edges.
     ConnectParentToAllParents(parent_subsplit, added_edge_idxs);
   }
-  // Create reindexers.
-  node_reindexer = BuildNodeReindexer(prev_node_count);
-  edge_reindexer = BuildEdgeReindexer(prev_edge_count);
-  // Update the ids in added_node_ids and added_edge_idxs according to the reindexers.
-  Reindexer::RemapIdVector(added_node_ids, node_reindexer);
-  Reindexer::RemapIdVector(added_edge_idxs, edge_reindexer);
-  // Update fields in the Subsplit DAG according to the reindexers.
-  RemapNodeIds(node_reindexer);
-  RemapEdgeIdxs(edge_reindexer);
-  // Recount topologies to update `topology_count_below_`.
-  CountTopologies();
+  if (!storage_.HavePrepend()) {
```

Comment:
So we don't reindex at all if we are in the graft DAG setting. This should at least be documented.

Reply:
Correct, that's the behavior of the previous GraftDAG implementation as well. Reindexing would modify the host dag.